android/assets/jsons/Civ V - Vanilla/UnitPromotions.json

```diff
@@ -227,19 +227,19 @@
 	// Aircraft Carrier
 	{
 		"name": "Armor Plating I",
-		"effect": "+25% Combat Bonus when defending",
```

Comment:
If w're unifying the text so the attacker and defender texts are imilar, I think the better way to do it is the opposite - keep "+[]% Strength when defending" and change the "Bonus as attacker" to "+[]% Strength when attacking"

Reply:
> 
> 
> If w're unifying the text so the attacker and defender texts are imilar, I think the better way to do it is the opposite - keep "+[]% Strength when defending" and change the "Bonus as attacker" to "+[]% Strength when attacking"

-Attacker Bonus unique changed to "+[]% Strength when attacking"
Changed Units.json and and Wolf Pack from UnitPromotions.json to this syntax

-Defender Bonus unique changed to "+[]% Strength when defending" 
Armor Plating promotions were changed to this syntax

Reply:
This may be just my OCD speaking, but may I suggest lowercase s for Strength? The sentence begins with the `+[]%` :rofl:

Reply:
Strength, like all values in gane (Range, Production etc), are consistently capitalized throughout the game